pycbc/waveform/waveform.py

```diff
@@ -927,6 +927,36 @@ def get_waveform_filter_length_in_time(approximant, template=None, **kwargs):
     else:
         return None
 
+def get_waveform_from_ascii(waveform,epoch=0.0,Norm=5.9029581035870565e+20):
```

Comment:
Can you make sure to follow PEP8? That means spaces after commas, and use only lower case variable names.